src/CardList/index.js

```diff
@@ -384,11 +383,11 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
   },
   rightMedia: {
-    height: 80,
-    width: 80,
-    marginTop: 4,
-    marginBottom: 8,
+    height: 100,
+    width: 100,
+    marginBottom: -25,
     marginLeft: 16,
+    marginRight: -16,
```

Comment:
Per Ben's response, we don't need to set the margin right here.

```suggestion
```

or

```suggestion
    marginRight: 0,
```